modules/react/common/lib/CanvasProvider.tsx

```diff
@@ -1,34 +1,60 @@
 import * as React from 'react';
 import {Theme, ThemeProvider} from '@emotion/react';
 import {InputProvider} from './InputProvider';
-import {defaultCanvasTheme, PartialEmotionCanvasTheme} from './theming';
+import {defaultCanvasTheme, PartialEmotionCanvasTheme, useTheme} from './theming';
+import {brand} from '@workday/canvas-tokens-web';
 
 export interface CanvasProviderProps {
   theme: PartialEmotionCanvasTheme;
 }
 
-export class CanvasProvider extends React.Component<
-  CanvasProviderProps & React.HTMLAttributes<HTMLElement>
-> {
-  static defaultProps = {
-    theme: {
-      canvas: defaultCanvasTheme,
-    },
-  };
+const mappedKeys = {
+  lightest: 'lightest',
+  light: 'light',
+  main: 'base',
+  dark: 'dark',
+  darkest: 'darkest',
+  contrast: 'accent',
+};
```

Comment:
is this something that consumer will have to do if they're using canvas provider?

Reply:
They shouldn't have too. We're just mapping the existing structure to the new structure.